<ui version="4.0">
 <class>qMRMLMarkupsLabelWidget</class>
 <widget class="QWidget" name="qMRMLMarkupsLabelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>表单</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkCollapsibleButton" name="labelCollapsibleButton">
     <property name="text">
      <string>标签设置:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelLabel">
        <property name="text">
         <string>标签:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="labelTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="plainText">
         <string />
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ThreeDTipDimensionModeLabel">
        <property name="text">
         <string>3D 提示维度模式：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="ThreeDTipDimensionModeComboBox">
        <property name="toolTip">
         <string>调整 3D 视图中的箭头提示尺寸，以改善表示。\n\n视图缩放因子：提示的高度是每像素毫米视图缩放因子的 6 倍，\n线长度：提示的高度是线长度的 3%，\n固定：提示的高度为 1.0 毫米。\n\n提示的高度始终是提示半径的 3 倍。</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources />
 <connections />
</ui>